### ISSUE TITLE
Higher top1 in SeAct dataset

### ISSUE THREAD

**Issue by Zizzzzzzz** [jiazhou-garland/ExACT#15]
On the SeAct dataset, I used the default sample_event_num_min of 100,000 and sample_event_threshold of 40 to generate AFE.
I downloaded the CLIP weights from "https://openaipublic.azureedge.net/clip/models/5806e77cd80f8b59890b7e101eabd078d9fb84e6937f9e85e4ecb61988df416f/ViT-B-16.pt". 
Since some sequences only generated a single frame of AFE, I modified line 29 in SeAct_sampled_dataset.py 
`from for i in range(len(event_stream_path)-1):`
to
`for i in range(len(event_stream_path)):`
Ultimately, this resulted in a top-1 accuracy of 81.03%.

![Image](https://github.com/user-attachments/assets/a0ef26b7-63a1-4033-bb87-8f3870b9efc1)

**Comment by jiazhou-garland** [comment]
Please adjust the default sample_event_num_min and sample_event_threshold to generate AFE until all sequences produce a reasonable number of frames.

**Comment by jiazhou-garland** [comment]
Some sequences that generated only a single frame of AFE are clearly undersampled.

**Comment by Zizzzzzzz** [comment]
Could you please provide some specific adjustment suggestions?

> Some sequences that generated only a single frame of AFE are clearly undersampled.

**Comment by Zizzzzzzz** [comment]
What do you think is the reason for the higher accuracy in this experiment compared to the paper? Could there be an issue with the CLIP weights I downloaded?

**Comment by Zizzzzzzz** [comment]
> What do you think is the reason for the higher accuracy in this experiment compared to the paper? Could there be an issue with the CLIP weights I downloaded?

I found that the sequences in SeAct_val.txt and SeAct_sampled_val.txt don't match. Some sequences from SeAct_val.txt appear in SeAct_sampled_train.txt. When I use SeAct_val.txt for testing, this results in some test sequences containing data from the training set.
